kubernetes/ama-logs.yaml

```diff
@@ -384,7 +384,7 @@ spec:
         #         - NET_ADMIN
         #         - NET_RAW
         - name: ama-logs
-          image: "mcr.microsoft.com/azuremonitor/containerinsights/ciprod:ciprod10042022-3c05dd1b"
+          image: "mcr.microsoft.com/azuremonitor/containerinsights/ciprod:ciprod11292022-6a507f9a"
           imagePullPolicy: IfNotPresent
           resources:
```

Comment:
can you please update from master.220318.3 to  master.221118.2 in the ama-logs.yaml since this latest addon-token-adapter image in aks side.

Reply:
updated